code/modules/urist/modules/shipbattles/weapons.dm

```diff
@@ -282,6 +282,119 @@
 	desc = "It's a big warhead for a big torpedo. Shove it in a torpedo casing and you've got yourself a torpedo." //torpedo
 	icon = 'icons/urist/items/ship_projectiles.dmi'
 	icon_state = "torpedowarhead"
+	var/safety = 1 //Integrated safeties are functioning as intended. This thing's inert.
+	var/is_rigged = 0 //Has this thing been jerry-rigged?
+	var/riggedstate = 0 //What stage of rigging are we in?
```

Comment:
While not required you may want to use defines for these states.

Reply:
Seems somewhat silly to use defines for these.

Reply:
Just makes it more readable by having `if(riggedstate == PANEL_OPEN)`,etc. You would only have them defined for this file alone.

Reply:
In the next commit.